engine/schema/src/main/java/com/cloud/vm/AllowedConsoleSessionVo.java

```diff
@@ -0,0 +1,64 @@
+//
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+//
+
+
+package com.cloud.vm;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "allowed_console_session")
+public class AllowedConsoleSessionVo {
```

Comment:
Can this be just `console_session` and `ConsoleSessionVO`?

Reply:
@GutoVeronezi what do you think of reusing the console_proxy table for this (there is a column already for this: `session_details`) instead of a new table?

Reply:
Agree, if this is just one column we need that we think all VMs should have, reusing existing table makes sense.

Reply:
LGTM

Reply:
> @DaanHoogland @nvazquez @rohityadavcloud
> 
> I forgot to mention it in the description. My idea while implementing this way was to extend it in the future to store more data (when/which user generated the console session, when/which IP consumed it, and so on - indeed, `console_session` would be a better name), to allow operators to track the console sessions; And the entries would be marked as removed/used instead of being deleted.
> 
> As gathering those data would require a more accurate analysis of the console session workflow, I first implemented this way to handle the situation; then I would improve the feature.
> 
> What do you think?

@GutoVeronezi 
make sense. can you please add created/removed columns and VOs ? it seems simple.
please also use `remove` instead of `expunge` when you remove a session.